tests/test_recipient_validation.py

```diff
@@ -8,90 +8,70 @@
     format_recipient,
     get_international_phone_info,
     international_phone_info,
-    is_uk_phone_number,
-    normalise_phone_number,
+    is_us_phone_number,
     try_validate_and_format_phone_number,
     validate_and_format_phone_number,
     validate_email_address,
     validate_phone_number,
 )
 
-valid_uk_phone_numbers = [
-    '7123456789',
-    '07123456789',
-    '07123 456789',
-    '07123-456-789',
-    '00447123456789',
-    '00 44 7123456789',
-    '+447123456789',
-    '+44 7123 456 789',
-    '+44 (0)7123 456 789',
-    '\u200B\t\t+44 (0)7123 \uFEFF 456 789 \r\n',
+valid_us_phone_numbers = [
+    '1-202-555-0104',
+    '+12025550104',
+    '12025550104',
+    '2025550104',
+    '(202) 555-0104',
 ]
 
 
 valid_international_phone_numbers = [
-    '71234567890',  # Russia
-    '1-202-555-0104',  # USA
-    '+12025550104',  # USA
-    '0012025550104',  # USA
-    '+0012025550104',  # USA
-    '23051234567',  # Mauritius,
+    '+71234567890',  # Russia
+    '+447123456789',  # UK
+    '+4407123456789',  # UK
+    '+4407123 456789',  # UK
+    '+4407123-456-789',  # UK
+    '+23051234567',  # Mauritius,
     '+682 12345',  # Cook islands
     '+3312345678',
-    '003312345678',
-    '1-2345-12345-12345',  # 15 digits
+    '+9-2345-12345-12345',  # 15 digits
 ]
 
 
-valid_phone_numbers = valid_uk_phone_numbers + valid_international_phone_numbers
+valid_phone_numbers = valid_us_phone_numbers + valid_international_phone_numbers
 
 
-invalid_uk_phone_numbers = sum([
+invalid_us_phone_numbers = sum([
     [
         (phone_number, error) for phone_number in group
     ] for error, group in [
         ('Too many digits', (
-            '712345678910',
-            '0712345678910',
-            '0044712345678910',
-            '0044712345678910',
-            '+44 (0)7123 456 789 10',
+            '55512345678',
+            '+155512345678',
+            '(555) 1234-5678',
         )),
         ('Not enough digits', (
-            '0712345678',
-            '004471234567',
-            '00447123456',
-            '+44 (0)7123 456 78',
+            '555123123',
+            '(555) 123-123',
+            '7890x32109',
+            '07123 ☟☜⬇⬆☞☝',
+            '07123☟☜⬇⬆☞☝',
```

Comment:
Why do you think these are parsing as a possible phone number? That seems unexpected to me.

Reply:
Non-numbers are basically stripped out. From the [`parse` docs](https://github.com/daviddrysdale/python-phonenumbers/blob/b3e8f976a11954564190266c50bac33b5721b66e/python/phonenumbers/phonenumberutil.py#L2855-L2863):

> The method is quite lenient and looks for a number in the input text
> (raw input) and does not check whether the string is definitely only a
> phone number. To do this, it ignores punctuation and white-space, as
> well as any text before the number (e.g. a leading "Tel: ") and trims
> the non-number bits.  It will accept a number in any format (E164,
> national, international etc), assuming it can be interpreted with the
> defaultRegion supplied. It also attempts to convert any alpha characters
> into digits if it thinks this is a vanity number of the type "1800
> MICROSOFT".